### PR TITLE
Speed up dragging animation to be at least 60fps

### PR DESCRIPTION
**Pull request by special-character**
Update the throttle to allow dragging to be at least 60fps.

Update Animated.timing values to `useNativeDriver: true`

### CODE REVIEW

File: src/DraxView.tsx [hunk 1]
```diff
@@ -392,7 +392,7 @@ export const DraxView = (
 				// Pass the event up to the Drax context.
 				handleGestureEvent(id, event);
 			},
-			33,
+			10,
```
Comment (special-character) [inline src/DraxView.tsx:395]: This allows dragging to be at least 60 fps. 

i.e. 1000ms/33ms = 30.3fps

Now 100fps max

**Comment by lafiosca** [comment]
@special-character Have you tested this change and found it to perform the way you were hoping/expecting? I don't recall this part of the code very well, but I believe I put in the throttling because too many events were spamming and impacting performance.

**Comment by special-character** [comment]
@lafiosca I did test this by dragging an item in a long list. I found that the item I was dragging would fall behind my finger/drag point and felt a little slow. After making this update it felt faster and didn't fall behind. 

I didn't see any perf issues after that so I think it might be a good level of throttling. I didn't see any perf issues with the `Animated.timing` events but I figured using the native driver couldn't hurt and might help if a lot of animation was happening.

**Comment by lafiosca** [comment]
I'm going to attempt to set up my new environment for React Native development tonight and dig into the code again a bit, and I'll try out these changes and see if they seem ok on my end as well. In fact, I will try working on this during my Twitch stream this evening which should start roughly around 7pm eastern US time, if you have any interest in joining: https://www.twitch.tv/lafiosca

**Comment by lafiosca** [comment]
I didn't get around to looking at this one tonight, but I will try to address it on Thursday.

**Comment by lafiosca** [comment]
Sorry for the delay again, I'll try to tackle it this weekend.

**Comment by special-character** [comment]
@lafiosca No need to be sorry! We are doing this in our fork but we can update whenever (or help out on things that come up)

**Comment by lafiosca** [comment]
I was able to test this on iOS Simulator and Android emulator. In the iOS Simulator I did not see any significant performance penalty from your change, so I feel ok about merging it there. In the Android emulator, both the old and new versions are extraordinarily slow (gesture input handling lag) on my new system, which is surprising. Unfortunately my physical Android test device seems to have decided it no longer wants to function. I am going to cross my fingers that this change has not introduced any performance issues on Android, and I'll merge it. I'm noting this here in case we run into problems when I do have a physical Android device to test with once again. Thank you for the PR.

**Pull request merged**